cmd/profiles.go

```diff
@@ -235,3 +237,57 @@ func availableProfilesAsAList() ([]string, error) {
 
 	return profileNames, nil
 }
+
+func getProfileFlag(cmd *cobra.Command) (*profile.Profile, error) {
```

Comment:
Moved this function and `getProviderFromProfile` function from `cmd/stack.go` into this one (cmd/profiles.go), since they look more related to profiles. But it's true that in this file it is not used...

This function will be used also in `cmd/testrunner.go` to get/load the profile to be used in tests.